tests/foreman/cli/test_errata.py

```diff
@@ -492,6 +485,62 @@ def test_positive_install_by_host_collection_and_org(
 
 @pytest.mark.tier3
 def test_negative_install_erratum_on_host_collection(
+    module_entitlement_manifest_org, host_collection, errata_hosts, target_sat
```

Comment:
There are a number of apparent test additions in this file, is this a history conflict?

Reply:
some conflicts are resolved now. rebase with master